src/elements/Button/Button.spec.tsx

```diff
@@ -51,6 +51,28 @@ describe("Button component", () => {
         });
     });
 
+    it(`renders in solid Primary inverted with only text.`, () => {
```

Comment:
You have the same test twice. Maybe add a data-test-id that reflects if the button is inverted or check if the inverted styles are correct.